hedera-node/docs/design/services/smart-contract-service/accounts-blocklisting.md

```diff
@@ -0,0 +1,68 @@
+# Accounts Blocklisting
+
+## Purpose
+
+In Ethereum, everyone that knows a private key is able to authorize a transaction from the corresponding EVM public address, and the first to send a transaction would be eligible to withdraw any funds from that address.
+In Hedera, it is possible for an account with associated EVM address alias to have a key that does not correspond to the alias. This is what we refer to as “EVM address squatting” i.e. using a key that’s different from the key that’s used to derive the EVM address alias from.
```

Comment:
I wouldn't term this as such given it's a likely occurrence of safe key practices i.e. key rotation.
I think the squatting here is the act of creating account with matching evm address to publicly known exposed private keys - that's the act of squatting.

Reply:
Updated.